qiskit/wrapper/configrc/_configrc.py

```diff
@@ -0,0 +1,268 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+"""
+A module for generating and manipulating the qiskitrc file.
```

Comment:
I originally did that but it is not very clean, and the minute we add other stuff to the file we run into trouble.  Since we can store a dict, that seemed to be the cleanest way.

Reply:
It is preferable to have a default `.qiskitrc` file with a template discussing the options and syntax rather than forcing the user to take a look at the code.

Reply:
Again, I do not want them to see the file.  I just want them to use the provided functions to register.